mackup/applications/spacelaucher.cfg

```diff
@@ -0,0 +1,4 @@
+[application]
+name = spacelaucher
+[configuration_files]
+Library/Application Support/name.guoc.SpaceLauncher 
```

Comment:
trim the ending whitespace and end the line with a newline char

Reply:
Fixed in https://github.com/lra/mackup/pull/1356/commits/408d6739e2b21260d36e3fe0cfe2c5c04a9de3fd and https://github.com/lra/mackup/pull/1356/commits/1ceb51820cd714822605f2a3770ef9654dd7e6e6